lib/helpers.js

```diff
@@ -32,8 +32,8 @@ async function getDirectories (rootPath) {
 async function getAndroidPlatformAndPath () {
   const androidHome = process.env.ANDROID_HOME;
   if (!_.isString(androidHome)) {
-    log.error("ANDROID_HOME was not exported!");
-    return null;
+    log.error("ANDROID_HOME environment variable was not exported!");
+    return {platform: null, platformPath: null};
```

Comment:
Can we throw the error here immediately instead of retuning null values? Further testing does not have much sense if this verification fails